lang/java/reef-webserver/src/main/java/org/apache/reef/webserver/PortNumber.java

```diff
@@ -25,7 +25,11 @@
  * port number for the Http Server
  * Clients can set a preferred port number. However, Reef will detect if the given port number has been used
  * in the machine. If yes, it will generate another available port number.
+ * @deprecated "Use TcpPortRangeStart and TcpPortProvider instead."
```

Comment:
Please add the version of REEF in which this was deprecated. Also, in all of these, please drop the `"`, as JavaDoc would put it into the documentation verbatim.